newIDE/app/src/GameDashboard/LeaderboardAdmin/index.js

```diff
@@ -0,0 +1,820 @@
+              // $FlowFixMe
+              playerUnicityDisplayChoice: value,
+            });
+          }}
+          disabled={isRequestPending || isEditingName}
+          inputStyle={{ fontSize: '0.875em' }}
+          helperMarkdownText={
+            currentLeaderboard.playerUnicityDisplayChoice === 'FREE'
+              ? i18n._(
+                  t`Users can chose to see only players' best entries or not.`
+                )
+              : currentLeaderboard.playerUnicityDisplayChoice ===
+                'PREFER_UNIQUE'
+              ? i18n._(t`Only player's best entries are displayed.`)
+              : i18n._(t`All entries are displayed.`)
+          }
+        >
+          {[
+            <SelectOption
+              key={'free'}
+              value={'FREE'}
+              primaryText={t`Let the user select`}
+            />,
+            <SelectOption
+              key={'prefer-unique'}
+              value={'PREFER_UNIQUE'}
+              primaryText={t`Only best entry`}
+            />,
+            <SelectOption
+              key={'prefer-non-unique'}
+              value={'PREFER_NON_UNIQUE'}
+              primaryText={t`All entries`}
+            />,
+          ]}
+        </SelectField>
+      ),
+      secondaryText:
+        apiError &&
+        apiError.action === 'leaderboardPlayerUnicityDisplayChoiceUpdate' ? (
+          <Typography color="error" variant="body2">
+            {apiError.message}
+          </Typography>
+        ) : null,
+      secondaryAction: null,
+    },
+  ];
+
+  return (
+    <I18n>
+      {({ i18n }) => (
+        <ResponsiveLineStackLayout noMargin expand noColumnMargin>
+          <div style={styles.leftColumn}>
+            <Paper elevation={5} style={{ padding: 5, margin: 5 }}>
+              <Column>
+                <Line>
+                  <Autocomplete
+                    autoComplete
+                    blurOnSelect
+                    disableClearable
+                    noOptionsText={<Trans>No matching leaderboard</Trans>}
+                    style={{ flex: 1 }}
+                    options={leaderboards}
+                    getOptionLabel={option => option.name}
+                    onChange={(e, leaderboard) => {
+                      if (leaderboard) selectLeaderboard(leaderboard.id);
+                    }}
+                    getOptionSelected={(leaderboard, selectedLeaderboard) =>
+                      leaderboard.id === selectedLeaderboard.id
+                    }
+                    value={currentLeaderboard}
+                    renderInput={params => (
+                      <MUITextField
+                        {...params}
+                        margin="dense"
+                        label={<Trans>Leaderboard name</Trans>}
+                        variant="filled"
+                      />
+                    )}
+                  />
+                  <IconButton
+                    onClick={_createLeaderboard}
```

Comment:
I'm not a big fan of naming functions with an underscore - they are not private or anything, they are just local variables. Can you name these "onCreateLeaderboard" to show that it's what is called to react to a semantic event from the user?

Reply:
True, it feels better with the `on` instead of the underscores.